app/about/page.jsx

```diff
@@ -63,7 +63,7 @@ export default function About() {
               an equal opportunity to rise to the top of search results.
               </p>
             </div>
-            <div className='text-gray-700 text-center py-5 mx-5'>
+            <div className='text-black text-center py-5 mx-5'>
```

Comment:
why not adding the text-black in your h1 and not in your dev

Reply:
because I am targeting the whole div which includes the h1 and p tags

Reply:
i don't think that was the approach we agreed